io.embrace.sdk/Scripts/EmbraceStartupArgs.cs

```diff
@@ -0,0 +1,20 @@
+namespace EmbraceSDK.Internal
+{
+    public struct EmbraceStartupArgs
```

Comment:
Would instinctively prefer a `class` for anything in our public API as it makes it easier to tweak properties later without breaking things - but I'm not super familiar with C#. What are the main benefits of using a struct here?

Reply:
The primary argument here is that structs (in many cases) are stack copy objects rather than heap objects. This is a minor optimization in the grand scheme of things, but does slightly improve versus GC.

Additionally I am unsure if the use case for tweaking properties later is terribly useful in many cases. Unless they want to spin down and spin up the SDK again?

Reply:
I didn't word that very well - my understanding is that using a class rather than a struct makes it easier for us to make additive changes a few months down the line. For example, if we want to add a new `Foo` property that is optional, that's relatively straightforward with a class, but a breaking change with a struct.

(I'm assuming that C# structs work similarly to C structs, which may be wrong)

Reply:
Ah. That makes sense then. I'll change it to a class.